src/coreclr/src/ToolBox/superpmi/superpmi-shared/compileresult.cpp

```diff
@@ -43,11 +41,6 @@ CompileResult::~CompileResult()
 
     if (CallTargetTypes != nullptr)
         delete CallTargetTypes;
-
-#ifndef TARGET_UNIX // PAL doesn't have HeapDestroy()
-    if (codeHeap != nullptr)
```

Comment:
@dotnet/jit-contrib superpmi allocated a special heap for code on Windows, but not on Unix. I have unified it to not use the special heap everywhere. Any concerns about this?